opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/AnalyzerGuruHelp.java

```diff
@@ -48,7 +49,18 @@ private AnalyzerGuruHelp() {
      */
     public static String getUsage() {
         StringBuilder b = new StringBuilder();
-        b.append("AnalyzerGuru prefixes:\n");
+
+        b.append("List of analyzers:\n");
+        b.append("The names of the analyzers (left column) can be used for the -A indexer option:\n\n");
+        byFactory(AnalyzerGuru.getAnalyzerFactories().stream().
+                collect(Collectors.toMap(f -> f.getClass().getSimpleName(), f -> f))).
+                forEach((factory) -> {
+            b.append(String.format("%-10s : %s\n",
```

Comment:
Just a nitpick, even though it should work fine on Windows, maybe it would be better to use `System.lineSeparator()`?

Btw, I think that with with `String.format` you can use `%n`.